portfolio/src/main/webapp/index.html

```diff
@@ -173,6 +173,21 @@ <h4 class="text-center">Post your thoughts!</h4>
             <button class="btn btn-primary" type="submit">Post</button>
           </form>
         </div>
+        <div class="card p-4 mb-4">
+          <form class="form-inline" id="comment-control">
+            <div class="form-group">
+              <button class="btn btn-primary" name="refresh">Refresh</button>
```

Comment:
If possible use HTML attributes to define which javascript functions elements call when events are triggered.
See comments.js for more detailed explanation.

```suggestion
              <button class="btn btn-primary" name="refresh" onclick="refreshComments()">Refresh</button>
```